styles.css

```diff
@@ -310,3 +310,73 @@ ul.technologies_container-languages > li > p {
 .border_bottom {
   border-bottom: 2px solid #dfe1e6;
 }
+
+/* contact form */
+article.contact_me {
+  background-image: url(images/contact_form_shapes-mobile@2x.png);
+  background-size: 98% 92%;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: auto;
+  margin-top: 3rem;
+  border-top-left-radius: var(--radiusL);
+  background-color: var(--primaryb400);
+  padding: 5rem 20px 20px 20px;
+}
+
+article.contact_me h2 {
+  text-align: center;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: var(--white);
+}
+
+.form {
+  display: grid;
+  grid-template-columns: auto;
+  gap: 1rem;
+  border: none;
+}
+
+input[type=text],
+input[type=email],
+input[type=submit],
+input[type=textarea] {
+  width: 100%;
+  padding: 12px;
+  border: none;
+  border-radius: 8px;
+  outline: none;
+}
+
+textarea {
+  padding: 20px;
+  background-color: var(--white);
+  outline: none;
+  border: none;
+  resize: none;
+  border-radius: 8px;
+  text-align: left;
+}
+
+.form_btn {
+  max-width: 120px;
+  padding: 3px 14px;
+  margin-top: -2.5rem;
+  margin-bottom: 2rem;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: var(--primaryb400);
+}
```

Comment:
- [ ] The `background-color` of the button should be `#fff`. Currently, the `background-color` is set by default colour HTML allocates to the `input` element with type `submit`. Kindly change the background-color to `#fff`. 👍 💻